README.md

```diff
@@ -53,6 +53,7 @@ $ ./artisan vendor:publish --provider="AvtoDev\IDEntity\ServiceProvider"
 `BODY`    | Номер кузова транспортного средства                            | `\AvtoDev\IDEntity\Types\IDEntityBody`
 `CHASSIS` | Номер шасси транспортного средства                             | `\AvtoDev\IDEntity\Types\IDEntityChassis`
 `DLN`     | Номер водительского удостоверения                              | `\AvtoDev\IDEntity\Types\IDEntityDriverLicenseNumber`
+`CAD    ` | Кадастровый номер объекта недвижимости                         | `\AvtoDev\IDEntity\Types\IDEntityCadastralNumber`
```

Comment:
`$ composer require avto-dev/identity-laravel "^4.1"`

Reply:
Maybe I should do this *sh\*t* in `faker-providers`